### DISCUSSION TITLE
How to Add or Update new CPE in the exist Internal Vulnerability

### DISCUSSION THREAD

**Discussion by JN-CSIRT** [DependencyTrack/dependency-track#2787] [Q&A]
Just created internal Vulnerability but do not add CPE/PURL, Now try to update it - add correct CPE, but it does not work?
how can I update CPE in this created Vulnerability?
Basically I can change/add everything else but not CPE...

**Comment by syalioune** [comment]
Hello @JN-CSIRT 
Once the internal vulnerability is created, it is not possible to update the related vulnerable components (i.e. CPE, PURL, ...).
You can file an enhancement request for this feature.
